addons/l10n_cl/views/report_invoice.xml

```diff
@@ -208,9 +213,22 @@
             <div class="row">
                 <div name="stamp" class="col-4 text-center"/>
                 <div name="transferable-table" class="col-4"/>
-                <div name="transferable-legend" class="col-4 float-end"/>
+                <div name="transferable-legend" class="col-4 text-end">
+                    <t t-if="o.l10n_latam_document_type_id.code == '39' and o.country_code == 'CL'">
+                        The VAT tax of this boleta is: <span t-esc="o._l10n_cl_get_amounts()['vat_amount']" t-options="{'widget': 'monetary', 'display_currency': o.currency_id}"/>.
```

Comment:
total_amounts variable?

Reply:
Well, this is fixed, since now we call the method `_l10n_cl_get_amounts()` from `_l10n_cl_get_invoice_totals_for_report`